src/crepe/system/RenderSystem.cpp

```diff
@@ -30,17 +28,15 @@ void RenderSystem::update_camera() {
 }
 
 bool RenderSystem::render_particle(const Sprite & sprite,
-								   const Transform & tm) {
+								   Transform tm) {
```

Comment:
kun je deze tranform weghalen en hem maken in deze functie is netter

Reply:
??

Reply:
Je misbruikt transform